app.py

```diff
@@ -204,6 +205,24 @@ def process_liquidity(main_chart_data: pandas.DataFrame, collateral_token: str,
 
     return main_chart_data, collateral_token_price
 
+
+def create_accumulated_debt(data: pandas.DataFrame) -> pandas.DataFrame:
+    """Create a DataFrame with cumulative liquidable debt and flip the cumulative column."""
+    accumulated_debt = data[['collateral_token_price', 'liquidable_debt']].copy()
+    accumulated_debt['Cumulative_liquidable_debt'] = accumulated_debt['liquidable_debt'].cumsum()
+    # Flip the cumulative column
+    accumulated_debt['Cumulative_liquidable_debt'] = accumulated_debt['Cumulative_liquidable_debt'].iloc[::-1].values
```

Comment:
This wouldn't even work the right way! Luckily, you don't need to do the cumsum.